sympy/physics/units/tests/test_quantities.py

```diff
@@ -518,7 +518,8 @@ def test_issue_22164():
 
 
 def test_issue_22819():
-    from sympy.physics.units import tonne, gram
+    from sympy.physics.units import tonne, gram, Da
     from sympy.physics.units.systems.si import dimsys_SI
     assert tonne.convert_to(gram) == 1000000*gram
     assert dimsys_SI.get_dimensional_dependencies(area) == {'length': 2}
+    assert Da.scale_factor == 1.66053906660000e-24
```

Comment:
How does this pass?
```python
>>> Da = Quantity('dalton',abbrev="Da")
>>> Da.scale_factor
1
>>> Da.set_global_relative_scale_factor(amu/kilo, kg)
>>> Da.scale_factor
atomic_mass_constant
>>> amu.scale_factor
1.66053906660000e-24
>>> amu == 1.66053906660000e-24
False
>>> Da.scale_factor == 1.66053906660000e-24
False
```

Reply:
oh -- global scale vs quantity scale.

Reply:
> How does this pass?

Surprised that this didn't give an assertion error in this first place .How should this be dealt with ?

Reply:
But a quick look at other uses of global scale setting show args to be `(literal number, unit)`; it seems like it would be better to make the relative scale factor be `amu.scale_factor/kilo` and then the quantity scale setting would be unecessary? And it might be a good safeguard to do a check in global scale that the args are as expected: literal number, unit. So after `scale_factor = sympify(scale_factor)` we should verify that `scale_factor.is_number`.

Reply:
This needs to be checked. Is our amu the same as the dalton or are we distinguishing between the unified amu and the old amu? Is it right to define a new quantity -- or is Da just an alias for (u)amu?

Reply:
> This needs to be checked. Is our amu the same as the dalton or are we distinguishing between the unified amu and the old amu? Is it right to define a new quantity -- or is Da just an alias for (u)amu?

Yes ,this is why I had seperately defined dalton and not linked it with atomic_mass_constant earlier .Dalton was discovered much later than old amu and is referred as u(amu) .I believe old amu is much more of a universal constant than a unit.
https://en.m.wikipedia.org/wiki/Dalton_(unit)
https://chemistry.stackexchange.com/questions/62579/when-was-the-amu-renamed-the-dalton

Reply:
@smichr I would request for some assistance on how to deal with this .

Reply:
Our current definition for amu is using the same constant as you are proposing for Da so it seems to me that simply creating another alias for amu is in order: `Da = amu = atomic_mass_unit`. Maybe the old value could be added if necessary? `c12_amu`?

Reply:
Yes this seems a fine fix, also avoiding unnecessary confusion . `Da = amu = atomic_mass_unit`.


PS. I'm currently dealing with a fever ,I'll work on this as soon as I get better !

Reply:
Ping @smichr 
I have made the changes .Now Da is treated as an alias for amu . Thanks for the help and patience .